.github/data/matrix-smoke.json

```
@@ -1,90 +1,122 @@
 {
   "images": [
     {
+      "label": "ingresses 1/2",
       "image": "debian",
-      "marker": "ingresses"
+      "marker": "'ingresses and not annotations and not basic_auth and not hsts and not ingresses_smoke and not watch_namespace and not wildcard_tls'"
```

```suggestion
      "marker": "'ingresses and not annotations and not basic_auth and not hsts and not watch_namespace and not wildcard_tls'"
```
based on comment below